File: easy/8_number_pyramid_pattern/number_pyramid_pattern.go
package main

import "fmt"

func pyramidPattern(n int) {
	for i := 1; i <= n; i++ {
		for j := 0; j < n-i; j++ {
			fmt.Printf(" ")
		}
		for k := 0; k < i; k++ {
			fmt.Printf("%d", i)
		}
		fmt.Printf("\n")
	}
}

func main() {
	pyramidPattern(5)
}

File: easy/8_number_pyramid_pattern/number_pyramid_pattern.js
// Number Pyramid Pattern

function pyramidPattern(n) {
    for (let i = 1; i <= n; i++) {
        for (let j = 0; j < n-i; j++) {
            process.stdout.write(" ")
        }
        for (let k = 0; k < i; k++) {
            process.stdout.write(i.toString())
        }
        process.stdout.write("\n")
    }
}

pyramidPattern(5);

module.exports = pyramidPattern;
